trajopt_ifopt/src/constraints/collision/continuous_collision_evaluators.cpp

```diff
@@ -39,6 +39,10 @@ LVSContinuousCollisionEvaluator::LVSContinuousCollisionEvaluator(
   , collision_config_(std::move(collision_config))
   , dynamic_environment_(dynamic_environment)
 {
+  if (collision_config_->type != tesseract_collision::CollisionEvaluatorType::CONTINUOUS &&
```

Comment:
Is this check necessary? You don't do this in the LVSDiscreteCollisionEvaluator or the SingleTimestepCollisionEvaluator either. Not that it's bad to have an extra check, but for performance reasons. Or add the check everywhere.